package.yaml

```diff
@@ -27,6 +27,7 @@ dependencies:
 - unagi-chan >= 0.4.0.0 && < 0.5
 - async >= 2.2 && < 3
 - type-errors >= 0.2.0.0
+- type-errors-pretty >= 0.0.0.0 && < 0.1
```

Comment:
Is the lower bound here necessary if it's just going to be `0.0.0.0`? 

I would have thought that `type-errors-pretty < 0.1` would suffice.

Reply:
@symbiont-joseph-kachmar Usually I specify both bounds: lower and upper. This helps to have more robust build plans. Yes, I understand that `0.0.0.0` is the smallest possible version, so in this particular corner case it should be enough to specify `< 0.1`. I'm using `^>=` Cabal operator to specify lower and upper bounds at the same time. But I see that `.cabal` file here is generated via `package.yaml` and uses `cabal-version: 1.12` so I can't use this operator. That's why writing bounds explicitly not to forget to update them both in case of breaking changes.